### PR TITLE
update: add build details 1.6.20-M1 to eap.md

### PR DESCRIPTION
**Pull request by PetrakovichVictoria**

### CODE REVIEW

**Review by koshachy — APPROVED** [review]
LGTM

File: docs/topics/eap.md [hunk 1]
```diff
@@ -26,4 +26,40 @@ check [our instructions on how to configure your build to support this version](
...
+                <li>Multiplatform: hierarchical structure support enabled by default for all new multiplatform projects</li>
+                <li>CocoaPods Gradle plugin: customization of podspec properties, accepting the podspec version when configuring CocoaPods for Kotlin/Native, new tasks for generating a suitable podspec for XCFramework builds</li>
+                <li>Kotlin/JS: incremental compilation in the IR backend, lazy initialization of top-level properties by default, inline <code>Char</code> class, ability to use inline classes in external types, export improvements</li>
+                <li>Gradle: new flexible way of defining a Kotlin compiler execution strategy, removed <code>kotlin.parallel.tasks.in.project</code> build option, deprecated <code>kapt.use.worker.api</code> and <code>kotlin.coroutines</code> build options</li>
+            </ul>
+            <p>For more details, please refer to the <a href="https://github.com/JetBrains/kotlin/releases/tag/v1.6.0-M1">changelog</a>.</p>
```
Comment (olme04) [inline docs/topics/eap.md:50]:
```suggestion
            <p>For more details, please refer to the <a href="https://github.com/JetBrains/kotlin/releases/tag/v1.6.20-M1">changelog</a>.</p>
```

:)

Comment (PetrakovichVictoria) [reply docs/topics/eap.md]: Thanks!

File: docs/topics/eap.md [hunk 2]
```diff
@@ -26,4 +26,40 @@ check [our instructions on how to configure your build to support this version](
...
\ No newline at end of file
+<table>
+    <tr>
+        <th>Build info</th>
+        <th>Build highlights</th>
+        <th>Recommended kotlinx library versions</th>
+    </tr>
+    <tr>
+        <td><strong>1.6.20-M1</strong>
+            <p>Released: <strong>February 2, 2022</strong></p>
```
Comment (PetrakovichVictoria) [inline docs/topics/eap.md:37]:
```suggestion
            <p>Released: <strong>February 7, 2022</strong></p>
```

File: docs/topics/eap.md [hunk 3]
```diff
@@ -26,4 +26,40 @@ check [our instructions on how to configure your build to support this version](
...
+                <li><a href="https://github.com/Kotlin/kotlinx.coroutines" target="_blank"><strong>kotlinx.coroutines</strong></a> version: <a href="https://github.com/Kotlin/kotlinx.coroutines/releases/tag/1.6.0" target="_blank">1.6.0</a></li>
+                <li><a href="https://github.com/Kotlin/kotlinx.atomicfu" target="_blank"><strong>kotlinx.atomicfu</strong></a> version: <a href="https://github.com/Kotlin/kotlinx.atomicfu/releases/tag/0.17.0" target="_blank">0.17.0</a></li>          
+                <li><a href="https://github.com/Kotlin/kotlinx-datetime" target="_blank"><strong>kotlinx-datetime</strong></a> version: <a href="https://github.com/Kotlin/kotlinx-datetime/releases/tag/v0.3.2" target="_blank">0.3.2</a></li>          
+                <li><a href="https://ktor.io/" target="_blank"><strong>ktor</strong></a> version: <a href="https://github.com/ktorio/ktor/releases/tag/2.0.0-beta-1" target="_blank">2.0.0-beta-1</a></li>
```
Comment (PetrakovichVictoria) [inline docs/topics/eap.md:58]:
```suggestion
                <li><a href="https://ktor.io/" target="_blank"><strong>ktor</strong></a> version: <a href="https://github.com/ktorio/ktor/releases/tag/1.6.7" target="_blank">1.6.7</a></li>
```

Comment (PetrakovichVictoria) [reply docs/topics/eap.md:58]: Under the discussion

File: docs/topics/eap.md [hunk 4]
```diff
@@ -26,4 +26,40 @@ check [our instructions on how to configure your build to support this version](
...
\ No newline at end of file
+<table>
+    <tr>
+        <th>Build info</th>
+        <th>Build highlights</th>
+        <th>Recommended kotlinx library versions</th>
+    </tr>
+    <tr>
+        <td><strong>1.6.20-M1</strong>
+            <p>Released: <strong>February 7, 2022</strong></p>
```
Comment (PetrakovichVictoria) [inline docs/topics/eap.md:37]:
```suggestion
            <p>Released: <strong>February 8, 2022</strong></p>
```

**Pull request merged**